docs/getting-started.md

```diff
@@ -1,18 +1,144 @@
 # Getting Started
 
-Really rough getting started brain dump.
+This guide will show how to setup host machine for Yocto build and then how to build and boot ACRN Hypervisor, Service VM and User VMs on Intel platforms.
```

Comment:
"how to setup host machine" --> "how to set up the host machine" (the verb form is "set up"; the noun form is "setup")